open-craft/labxchange-xblocks
# Replace story in science with narrative xblock.

Renames Story in Science to Narrative.

Testing instructions:

Checkout branch in /src/
```
make studio-shell
pip install -e /edx/src/labxchange-xblocks/
exit
make restart-studio

make lms-shell
pip install -e /edx/src/labxchange-xblocks/
make restart lms
```

* In http://localhost:18010/settings/advanced/course-v1:edX+DemoX+Demo_Course, add `lx_narrative` to "Advanced Module List".
* Check that you are now able to add Narrative XBlocks to units.
* After publishing the XBlock renders correctly in LMS as well.
